CONTRIBUTING.md

```diff
@@ -0,0 +1,47 @@
+# How to contribute to Concuerror
+
+First of all, thank you for taking time to contribute to Concuerror!
+
+
+## I just have a question!
+
+Use the [mailing list][mailing list] and take a look at the [FAQ][faq] if you haven't already!
+
+
+## Did you encounter a bug?
+
+* **Ensure the bug has not already been reported** by searching Github's [Issues][Issues]. If you find a relevant issue, feel free to leave a new comment there.
+
+* If you cannot find an existing relevant Issue, [open a new Issue][open a new Issue]. Include a **title** and a **description**, as much relevant information as possible, and a **code sample** or an **executable test case** demonstrating the expected behavior that is not occurring.
+
+
+## Did you write a patch that fixes a bug?
+
+* Fork Concuerror's repository and push a new branch for your patch, using a good, descriptive name.
+
+* Open a Pull Request (PR) with the patch.
+
+* Ensure the PR's description explains the problem and solution. Include the relevant issue number, if applicable.
```

Comment:
Add "Make sure you include to your PR a test that shows that the bug is now fixed." here